app-backend/tile/Dockerfile

```diff
@@ -1,8 +1,16 @@
 FROM openjdk:8-jre-alpine
 
+ENV ASPECTJ_WEAVER_VERSION 1.8.10
```

Comment:
AspectJ Weaver is necessary for Kamon to initialize properly when invoked from an assembly JAR.

See: http://kamon.io/documentation/get-started/